### PR TITLE
fix: use GET method to launch Okta Verify

### PR DESCRIPTION
**Pull request by jhe-okta**
RESOLVES: OKTA-277738

Screen Shot 2020-02-14 at 7 50 02 AM

### CODE REVIEW

File: src/v2/view-builder/views/DeviceChallengePollView.js [hunk 1]
```diff
@@ -116,7 +116,6 @@ const Body = BaseForm.extend({
   doCustomURI () {
     return request({
       url: this.customURI,
-      method: 'POST',
```
Comment (haishengwu-okta) [inline src/v2/view-builder/views/DeviceChallengePollView.js]: shall we add `method` to the API response in order to make UI dumb ?
https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/idp/idx/data/identify-with-device-launch-authenticator.json#L70

Comment (jhe-okta) [reply src/v2/view-builder/views/DeviceChallengePollView.js]: Yea, that's mentioned in some discussion. But no decision yet. I can follow up later.

Comment (haishengwu-okta) [reply src/v2/view-builder/views/DeviceChallengePollView.js]: could you cut a JIRA and add to the code as comment?

Comment (jhe-okta) [reply src/v2/view-builder/views/DeviceChallengePollView.js]: https://oktainc.atlassian.net/browse/OKTA-278014

**Pull request merged**